ext/intl/formatter/formatter.stub.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+class NumberFormatter
+{
+    public function __construct(string $locale, int $style, ?string $pattern = null) {}
```

Comment:
`$pattern` doesn't seem to accept `null`:
```suggestion
    public function __construct(string $locale, int $style, string $pattern = '') {}
```
Same for `::create()` and `numfmt_create()`.

Reply:
Ah yes, I don't know what made me think that :)